Dockerfile

```diff
@@ -13,6 +14,8 @@ RUN luarocks-5.3 install dkjson && \
     luarocks-5.3 install luasocket && \
     luarocks-5.3 install multipart-post
 
+RUN apk del curl gcc libc-dev pcre-dev libressl-dev lua5.3-dev
```

Comment:
`curl` is required for uploads. Did you test this to make sure it works?

Reply:
I couldn't find any references to curl in the code, so I'm not sure what feature uses that, at least directly.

Reply:
Guess that's the case now. Just checked out the history and all the curl uses have been removed at one time or another.